index.js

```diff
@@ -161,35 +165,28 @@ exports.init = function (sbot, config) {
     }
   })
 
+  function findEBTForFeed(feedId) {
+    let ebt = Object.values(ebts).reverse().find(ebt => ebt.isFeed(feedId))
```

Comment:
The reverse() looks funny, I wonder if the next developer will find it confusing. Maybe we could use the previous `for`-loop we had before? That one was pretty clear to me.

Reply:
PS: good idea to drop the `formats` global and use just `ebts`